_episodes/04-recipe.md

```diff
@@ -635,58 +635,116 @@ The snippets for the edits can be found below:
 
 > ## Land surface average temperature
 >
```

Comment:
Maybe we can help the students understand the diff format a bit better then? 
```suggestion
>
> In the `diff` file below you will see the changes we have made to the file. The top 2 lines are the filenames and the lines like @@ -20,12 +20,14 @@ indicate the line numbers in the original and modified file, respectively. For more info on this format, see [here](https://en.wikipedia.org/wiki/Diff#Unified_format)
>
```

Reply:
Great suggestion